projects/ui/option/option.ts

```diff
@@ -147,7 +163,7 @@ export class CkOption<T = unknown> implements Highlightable, FocusableOption {
       return
     }
 
-    this.select()
+    this.toggle()
```

Comment:
Guess name of the parent method (`selectViaKeyboard`) should also be adjusted

Reply:
Done